lib/utils/getAcqUnitsOptions.js

```diff
@@ -1,5 +1,6 @@
-// eslint-disable-next-line import/prefer-default-export
-export const getAcqUnitsOptions = (acqUnits = []) => acqUnits.map(acqUnit => ({
-  value: acqUnit.id,
-  label: acqUnit.name,
-}));
+export const getAcqUnitsOptions = (acqUnits = []) => {
```

Comment:
the problem with default assignment is that `null` could be passed and it will not get the default value as empty array, so exception will rise. Maybe it makes sense to check for `Array.isArray` to call `map` only for arrays